src/Uplink/Admin/Actions.php

```diff
@@ -0,0 +1,95 @@
+<?php declare(strict_types=1);
+
+namespace StellarWP\Uplink\Admin;
+
+use StellarWP\Uplink\API;
+use StellarWP\Uplink\Config;
+use StellarWP\Uplink\Resources\Collection;
+use StellarWP\Uplink\Site\Data;
+
+class Actions {
+
+	const QUERY_VAR = 'stellarwp_action';
+
+	/**
+	 * @return void
+	 * @action init
+	 */
+	public function register_route() {
+		add_rewrite_endpoint( 'stellarwp', EP_ROOT, self::QUERY_VAR );
+	}
+
+	/**
+	 * @param \WP $wp
+	 *
+	 * @return void
+	 * @action parse_request
+	 */
+	public function handle_auth_request( $wp ) {
+		if ( empty( $wp->query_vars[ self::QUERY_VAR ] ) ) {
+			return;
+		}
+
+		$args = explode( '/', $wp->query_vars[ self::QUERY_VAR ] );
+
+		if (  ! empty( $args['disconnect'] ) ) { // @phpstan-ignore-line
```

Comment:
What's the error you are ignoring here?

Reply:
This is for `Offset 'disconnect' on non-empty-array<int, string> in isset() does not exist.`